src/Data/Nat/Divisibility/Core.agda

```diff
@@ -35,7 +35,27 @@ record _∣_ (m n : ℕ) : Set where
   constructor divides
   field quotient : ℕ
         equality : n ≡ quotient * m
-open _∣_ using (quotient) public
+
+  quotient≢0 : .{{NonZero n}} → NonZero quotient
+  quotient≢0 rewrite equality = m*n≢0⇒m≢0 quotient
+
+  n≡m*quotient : n ≡ m * quotient
+  n≡m*quotient rewrite equality = *-comm quotient m
+
+  quotient>1 :  (m<n : m < n) → 1 < quotient
+  quotient>1 m<n = ≰⇒> λ q≤1 → n≮n n (begin-strict
```

Comment:
There has got to be a cleaner way to prove this than the double negation used here. Sorry, I know that's not a very helpful comment!

Reply:
I know! (re: 'better' proofs) All of this came about in the context of #1969 #2180 #2181 where I had indeed found myself doing too much 'negative' reasoning (and rightly had that drawn to my attention in @Taneb 's review comments). 

As I had tagged this as v2.1, I thought that this could probably sit a while, but after our discussion of the v2.0 release/transition to v2.1, I realise that perhaps I should stop daydreaming. I definitely thought that another set of eyes might help with refactoring though ;-) and I share yours and Nathan's frustration about this style of reasoning... which also partly comes from not wanting to make the API of `Data.Nat.Properties` any wider, so trying to confine myself to using only what's already there...